modules/tracking/include/opencv2/tracking/tracker.hpp

```diff
@@ -1366,9 +1370,9 @@ class ROISelector {
 };
 
 Rect2d CV_EXPORTS_W selectROI(Mat img, bool fromCenter = true);
-Rect2d CV_EXPORTS_W selectROI(const cv::String& windowName, Mat img, bool showCrossair = true, bool fromCenter = true);
-void CV_EXPORTS_W selectROI(const cv::String& windowName, Mat img, std::vector<Rect2d> & boundingBox, bool fromCenter = true);
-
+Rect2d CV_EXPORTS_W selectROI(const cv::String &windowName, Mat img, bool showCrossair = true, bool fromCenter = true);
+void CV_EXPORTS_W selectROI(const cv::String &windowName, Mat img, std::vector<Rect2d> &boundingBox, bool fromCenter = true);
+void CV_EXPORTS_W selectROI(const cv::String &windowName, cv::VideoCapture &cap, std::vector<Rect2d> &boundingBox, bool fromCenter = true);
```

Comment:
@berak 
Thanks for your time.
I added the selecting with video running function so that the tracker doesn't have to search far away if the target is moving while user is selecting. If we select target on the static frame, the target may have already moved away when the tracking gets started.

I dont use python often so I didn't know the complain about passing a reference to VideoCapture.